app/build.gradle

```diff
@@ -10,8 +10,7 @@ apply from: '../config/quality/quality.gradle'
 android {
     compileSdkVersion rootProject.ext.compileSdkVersion
     buildToolsVersion rootProject.ext.buildToolsVersion
-    ndkVersion rootProject.ext.ndkVersion
-
+    
```

Comment:
This removal makes sense to me. I might have missed it, was in rush yesterday.

@rutvik-panchal 
- Can you clarify what was the reason behind the addition of this (since integrating workflows should be independent of the app)? (were you facing some issues, while testing it in your fork)
- Similarly, what's the reason for adding it to the workflow too? If possible, share some reference link.

Reply:
@miPlodder
 We only need to specify the ndk version either in the workflow file or the build.gradle files. And since it has already been specified in the workflow, so we don't need to specify it explicitly in the build.gradle files. Here is the reference for it: [Link](https://github.community/t/android-ndk-on-unix-is-incompatible-with-android-build-tools-3-6-0/17697). Hope this helps

Reply:
It's an [issue](https://github.community/t/android-ndk-on-unix-is-incompatible-with-android-build-tools-3-6-0/17697/13) and i have followed a comment ref. [link](https://github.community/t/android-ndk-on-unix-is-incompatible-with-android-build-tools-3-6-0/17697/15), If we are using gradle version < 4.1 then we need to specify the ndkVersion in build.gradle and we are currently at version 3.3.2 ([version code link](https://github.com/openMF/mifos-mobile/blob/d9fd53cd59a79cc33858133effc8e4f393a0bc4e/build.gradle#L10)).
I will check it once and let you know.

Reply:
We can remove it, I have tested on my forked repository [actions link](https://github.com/rutvik-panchal/mifos-mobile/actions/runs/389808022) for [changes](https://github.com/rutvik-panchal/mifos-mobile/commit/256ac30412ce4f087c101b86468dbe507f1af258).